docker.go

```diff
@@ -770,6 +771,25 @@ func NewDockerProvider(provOpts ...DockerProviderOption) (*DockerProvider, error
 		config:                tcConfig,
 	}
 
+	// log docker server info
```

Comment:
With current implementation of the library, this code will be executed with every container.

Could you make sure it's run just once? We are planning a big refactor in how the library is bootstrapped, see #549

Reply:
The first thing that comes to my mind is a simple bool check. But before I make that change, I wanted to ask you if you have any other suggestions/thoughts that inlines with #549.

Reply:
I have this at the tip of my head: you could enclose that block in a function, and call it using a sync.Once pattern. wdyt?

Reply:
Sounds better. Will do.